README.md

```diff
@@ -79,7 +79,7 @@ repositories {
 }
 
 dependencies {
-    testCompile "org.robolectric:robolectric:3.2-SNAPSHOT"
+    testCompile "org.robolectric:robolectric:3.2-gradle-SNAPSHOT"
```

Comment:
This is temporary, to prevent us clobbering snapshots until this is merged to master.